src/renderer/views/stake/StakeView.tsx

```diff
@@ -31,16 +32,7 @@ const StakeView: React.FC<Props> = (_) => {
 
   const keystore = useObservableState(keystoreService.keystore$, O.none)
 
-  const hasWallet = useMemo(
-    () =>
-      pipe(
-        keystore,
-        O.flatten,
-        O.map(Boolean),
-        O.getOrElse(() => false)
-      ),
-    [keystore]
-  )
+  const hasWallet = useMemo(() => pipe(keystore, isLocked), [keystore])
```

Comment:
I think it should be `!isLocked`, which means- an user has a wallet imported and it's not locked (full access to all wallet data). Or what's the idea behind "hasWallet"?